HikVisionSource/README.md

```diff
@@ -37,23 +37,16 @@ Make certain to use the correct implementation for the environment within which
 
 ## Prerequisite
 
-For windows based systems, one should download the Device Network SDK (for Windows 64 bit) from [*here*](https://www.hikvision.com/en/support/download/sdk/), extract the lib folder, set it access using an environment parameter. 
+For windows based systems, one should download the Device Network SDK (for Windows 64 bit or for Linux 64 bit) from [*here*](https://www.hikvision.com/en/support/download/sdk/), extract the lib folder, set it access using an environment parameter. 
```

Comment:
if you say Linux in the parentheses, then remove windows from the start.  Though this seems to repeat the line above, so maybe it could be deleted?
```suggestion
Download the Device Network SDK (for Windows 64 bit or for Linux 64 bit) from [*here*](https://www.hikvision.com/en/support/download/sdk/), extract the lib folder, set it access using an environment parameter. 
```

Reply:
done